packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1661,16 +1662,65 @@ export class StripeHelper extends StripeHelperBase {
     );
   }
 
-  getPaymentProvider(customer: Stripe.Customer) {
+  async getPaymentProvider(
+    customer: Stripe.Customer,
+    paymentIntentId?: string
+  ): Promise<SubPlatPaymentMethodType | 'paypal' | 'stripe' | 'not_chosen'> {
```

Comment:
`'stripe'` and `SubPlatPaymentMethodType.Stripe` are the same, so the extra return type shouldn't be needed here